### PR TITLE
DoughnutChart samples

### PR DESCRIPTION
**Pull request by igDivya**

### CODE REVIEW

File: samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        DataSource = dataItems;
+        this.onAnimationToggle();
+
+    }
+
+    private void OnTimerTick()
+    {
+        if (!this.isAnimating)
+        {
+            // Console.WriteLine("in");
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor:85]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            if (this.StartAngle < 360)
+                this.StartAngle++;
+
+            if (this.RadiusFactor >= 1.0 &&
+                this.StartAngle >= 360)
+            {
+                this.isAnimating = false;
+            }
+
+            //Console.WriteLine("OnTimerTick " + this.StartAngle);
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor:99]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+@code {
+
+    private List<DataItem> DataSource;
+	public int[] explodedSlices { get; set; } = new int[0];
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor:44]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    public class DataItem
+    {
+        public string Company { get; set; }
+        public double MarketShare { get; set; }
+        public string Summary { get { return Company + " " + MarketShare + "%"; } }
+    }
+
+    // NOTE here is example of even handler that update a field that is bound in HTML code above
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor:66]: remove comment

File: samples/charts/doughnut-chart/legend/Pages/DoughnutChartLegend.razor [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+</div>
+
+@code {
+
+    private List<DataItem> DataSource;
+   
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/legend/Pages/DoughnutChartLegend.razor:40]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/overview/Pages/DoughnutChartOverview.razor [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+</div>
+
+@code {
+
+    private List<DataItem> DataSource;
+   
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/overview/Pages/DoughnutChartOverview.razor:33]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor [hunk 7]
```diff
@@ -0,0 +1,106 @@
+﻿@page "/DoughnutChartRings"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igComponent">
+
+        @if (DataSource1 != null && DataSource2 != null)
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor:10]: rename `DataSource2` to `Seasons`
rename `DataSource1` to `Months`

File: samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+@code {
+
+    private List<Months> DataSource1;
+    private List<Seasons> DataSource2;
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor:46]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+        DoughnutChartModule.Register(InfragisticsBlazor);
+
+        //var dataItems = new List<DataItem>();
+        //dataItems.Add(new DataItem() { MarketShare = 30, Company = "Google", });
+        //dataItems.Add(new DataItem() { MarketShare = 25, Company = "Apple", });
+        //dataItems.Add(new DataItem() { MarketShare = 20, Company = "Microsoft", });
+        //dataItems.Add(new DataItem() { MarketShare = 15, Company = "Samsung", });
+        //dataItems.Add(new DataItem() { MarketShare = 10, Company = "Other", });
+
+        //DataSource = dataItems;
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor:57]: remove used code/comments: dataItems and DataSource

File: samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+        DataSource2 = seasons;
+
+    }
+
+    //public class DataItem
+    //{
+    //    public string Company { get; set; }
+    //    public double MarketShare { get; set; }
+    //    public string Summary { get { return Company + " " + MarketShare + "%"; } }
+    //}
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor:91]: remove unused classes

File: samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+
+    public class Months
+    {
+        public double  Value { get; set; }
+        public string Label { get; set; }
+        //public string Summary { get { return Company + " " + MarketShare + "%"; } }
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/rings/Pages/DoughnutChartRings.razor:97]: remove Summary  prop

File: samples/charts/doughnut-chart/selection/App.razor [hunk 12]
```diff
@@ -0,0 +1,3 @@
+@*NOTE uncomment one of samples:*@
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/selection/App.razor:1]: remove

File: samples/charts/doughnut-chart/selection/Pages/DoughnutChartSelection.razor [hunk 13]
```diff
@@ -0,0 +1,78 @@
+﻿@page "/DoughnutChartSelection"
+
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+@* selectedItem not working  *@
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/selection/Pages/DoughnutChartSelection.razor:8]: add `TODO` in comment

File: samples/charts/doughnut-chart/selection/Pages/DoughnutChartSelection.razor [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+@code {
+
+    private List<DataItem> DataSource;
+    private string SelectedSlices = "";
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        // Console.WriteLine("Sample runtime: " + InfragisticsBlazor.JsRuntime);
+        Console.WriteLine("Sample OnInitialized...");
```
Comment (HUSSAR-mtrela) [inline samples/charts/doughnut-chart/selection/Pages/DoughnutChartSelection.razor:48]: remove all Console.WriteLine()

File: samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor (hunk 1 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor]: done

File: samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor (hunk 2 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/animation/Pages/DoughnutChartAnimation.razor]: done

File: samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor (hunk 3 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor]: done

File: samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor (hunk 4 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/explosion/Pages/DoughnutChartExplosion.razor]: done

File: samples/charts/doughnut-chart/legend/Pages/DoughnutChartLegend.razor (hunk 5 above)
Comment (igDivya) [reply samples/charts/doughnut-chart/legend/Pages/DoughnutChartLegend.razor]: done

**Pull request merged**